03-Oct-2021 16:25:46.091 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextInitialized()
03-Oct-2021 16:25:46.092 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextInitialized()
03-Oct-2021 16:25:46.096 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: attributeAdded('StockTicker', 'async.Stockticker@6e0ff644')
03-Oct-2021 16:48:19.220 SEVERE [http-nio-8080-exec-18] org.apache.catalina.core.StandardWrapperValve.invoke El Servlet.service() para el servlet [jsp] en el contexto con ruta [] lanzó la excepción [Ha sucedido una excepción al procesar la página JSP [/peticionesRegistro.jsp] en línea [24]

21:         r.setEstado(Integer.parseInt(request.getParameter("estado")));
22:         r.setNombre_propietario(request.getParameter("nombre_propietario"));
23:         r.setDoc_propietario(Integer.parseInt(request.getParameter("doc_propietario")));
24:         r.setCel_propietario(Integer.parseInt(request.getParameter("cel_propietario")));
25:         r.setPlaca_vehiculo(request.getParameter("placa_vehiculo"));
26:         if(r.guardarRegistro()){
27:             respuesta += "\"" + proceso + "\": true";


Stacktrace:] con causa raíz
	java.lang.NumberFormatException: For input string: "123123345345"
		at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
		at java.base/java.lang.Integer.parseInt(Integer.java:652)
		at java.base/java.lang.Integer.parseInt(Integer.java:770)
		at org.apache.jsp.peticionesRegistro_jsp._jspService(peticionesRegistro_jsp.java:141)
		at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:777)
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:466)
		at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:380)
		at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:328)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:777)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:223)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:185)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:541)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:119)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:353)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:382)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:870)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1699)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:834)
03-Oct-2021 16:48:21.771 SEVERE [http-nio-8080-exec-22] org.apache.catalina.core.StandardWrapperValve.invoke El Servlet.service() para el servlet [jsp] en el contexto con ruta [] lanzó la excepción [Ha sucedido una excepción al procesar la página JSP [/peticionesRegistro.jsp] en línea [24]

21:         r.setEstado(Integer.parseInt(request.getParameter("estado")));
22:         r.setNombre_propietario(request.getParameter("nombre_propietario"));
23:         r.setDoc_propietario(Integer.parseInt(request.getParameter("doc_propietario")));
24:         r.setCel_propietario(Integer.parseInt(request.getParameter("cel_propietario")));
25:         r.setPlaca_vehiculo(request.getParameter("placa_vehiculo"));
26:         if(r.guardarRegistro()){
27:             respuesta += "\"" + proceso + "\": true";


Stacktrace:] con causa raíz
	java.lang.NumberFormatException: For input string: "123123345345"
		at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
		at java.base/java.lang.Integer.parseInt(Integer.java:652)
		at java.base/java.lang.Integer.parseInt(Integer.java:770)
		at org.apache.jsp.peticionesRegistro_jsp._jspService(peticionesRegistro_jsp.java:141)
		at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:777)
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:466)
		at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:380)
		at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:328)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:777)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:223)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:185)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:541)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:119)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:353)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:382)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:870)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1699)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:834)
03-Oct-2021 16:48:21.986 SEVERE [http-nio-8080-exec-23] org.apache.catalina.core.StandardWrapperValve.invoke El Servlet.service() para el servlet [jsp] en el contexto con ruta [] lanzó la excepción [Ha sucedido una excepción al procesar la página JSP [/peticionesRegistro.jsp] en línea [24]

21:         r.setEstado(Integer.parseInt(request.getParameter("estado")));
22:         r.setNombre_propietario(request.getParameter("nombre_propietario"));
23:         r.setDoc_propietario(Integer.parseInt(request.getParameter("doc_propietario")));
24:         r.setCel_propietario(Integer.parseInt(request.getParameter("cel_propietario")));
25:         r.setPlaca_vehiculo(request.getParameter("placa_vehiculo"));
26:         if(r.guardarRegistro()){
27:             respuesta += "\"" + proceso + "\": true";


Stacktrace:] con causa raíz
	java.lang.NumberFormatException: For input string: "123123345345"
		at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
		at java.base/java.lang.Integer.parseInt(Integer.java:652)
		at java.base/java.lang.Integer.parseInt(Integer.java:770)
		at org.apache.jsp.peticionesRegistro_jsp._jspService(peticionesRegistro_jsp.java:141)
		at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:777)
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:466)
		at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:380)
		at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:328)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:777)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:223)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:185)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:541)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:119)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:353)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:382)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:870)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1699)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:834)
03-Oct-2021 16:48:22.065 SEVERE [http-nio-8080-exec-24] org.apache.catalina.core.StandardWrapperValve.invoke El Servlet.service() para el servlet [jsp] en el contexto con ruta [] lanzó la excepción [Ha sucedido una excepción al procesar la página JSP [/peticionesRegistro.jsp] en línea [24]

21:         r.setEstado(Integer.parseInt(request.getParameter("estado")));
22:         r.setNombre_propietario(request.getParameter("nombre_propietario"));
23:         r.setDoc_propietario(Integer.parseInt(request.getParameter("doc_propietario")));
24:         r.setCel_propietario(Integer.parseInt(request.getParameter("cel_propietario")));
25:         r.setPlaca_vehiculo(request.getParameter("placa_vehiculo"));
26:         if(r.guardarRegistro()){
27:             respuesta += "\"" + proceso + "\": true";


Stacktrace:] con causa raíz
	java.lang.NumberFormatException: For input string: "123123345345"
		at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
		at java.base/java.lang.Integer.parseInt(Integer.java:652)
		at java.base/java.lang.Integer.parseInt(Integer.java:770)
		at org.apache.jsp.peticionesRegistro_jsp._jspService(peticionesRegistro_jsp.java:141)
		at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:777)
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:466)
		at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:380)
		at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:328)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:777)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:223)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:185)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:541)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:119)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:353)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:382)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:870)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1699)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:834)
03-Oct-2021 16:48:24.914 SEVERE [http-nio-8080-exec-25] org.apache.catalina.core.StandardWrapperValve.invoke El Servlet.service() para el servlet [jsp] en el contexto con ruta [] lanzó la excepción [Ha sucedido una excepción al procesar la página JSP [/peticionesRegistro.jsp] en línea [24]

21:         r.setEstado(Integer.parseInt(request.getParameter("estado")));
22:         r.setNombre_propietario(request.getParameter("nombre_propietario"));
23:         r.setDoc_propietario(Integer.parseInt(request.getParameter("doc_propietario")));
24:         r.setCel_propietario(Integer.parseInt(request.getParameter("cel_propietario")));
25:         r.setPlaca_vehiculo(request.getParameter("placa_vehiculo"));
26:         if(r.guardarRegistro()){
27:             respuesta += "\"" + proceso + "\": true";


Stacktrace:] con causa raíz
	java.lang.NumberFormatException: For input string: "123123345345"
		at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
		at java.base/java.lang.Integer.parseInt(Integer.java:652)
		at java.base/java.lang.Integer.parseInt(Integer.java:770)
		at org.apache.jsp.peticionesRegistro_jsp._jspService(peticionesRegistro_jsp.java:141)
		at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:777)
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:466)
		at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:380)
		at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:328)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:777)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:223)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:185)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:541)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:119)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:353)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:382)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:870)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1699)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:834)
03-Oct-2021 16:48:27.746 SEVERE [http-nio-8080-exec-19] org.apache.catalina.core.StandardWrapperValve.invoke El Servlet.service() para el servlet [jsp] en el contexto con ruta [] lanzó la excepción [Ha sucedido una excepción al procesar la página JSP [/peticionesRegistro.jsp] en línea [24]

21:         r.setEstado(Integer.parseInt(request.getParameter("estado")));
22:         r.setNombre_propietario(request.getParameter("nombre_propietario"));
23:         r.setDoc_propietario(Integer.parseInt(request.getParameter("doc_propietario")));
24:         r.setCel_propietario(Integer.parseInt(request.getParameter("cel_propietario")));
25:         r.setPlaca_vehiculo(request.getParameter("placa_vehiculo"));
26:         if(r.guardarRegistro()){
27:             respuesta += "\"" + proceso + "\": true";


Stacktrace:] con causa raíz
	java.lang.NumberFormatException: For input string: "123123345345"
		at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
		at java.base/java.lang.Integer.parseInt(Integer.java:652)
		at java.base/java.lang.Integer.parseInt(Integer.java:770)
		at org.apache.jsp.peticionesRegistro_jsp._jspService(peticionesRegistro_jsp.java:141)
		at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:777)
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:466)
		at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:380)
		at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:328)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:777)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:223)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:185)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:541)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:119)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:353)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:382)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:870)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1699)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:834)
03-Oct-2021 16:48:45.541 SEVERE [http-nio-8080-exec-20] org.apache.catalina.core.StandardWrapperValve.invoke El Servlet.service() para el servlet [jsp] en el contexto con ruta [] lanzó la excepción [Ha sucedido una excepción al procesar la página JSP [/peticionesRegistro.jsp] en línea [24]

21:         r.setEstado(Integer.parseInt(request.getParameter("estado")));
22:         r.setNombre_propietario(request.getParameter("nombre_propietario"));
23:         r.setDoc_propietario(Integer.parseInt(request.getParameter("doc_propietario")));
24:         r.setCel_propietario(Integer.parseInt(request.getParameter("cel_propietario")));
25:         r.setPlaca_vehiculo(request.getParameter("placa_vehiculo"));
26:         if(r.guardarRegistro()){
27:             respuesta += "\"" + proceso + "\": true";


Stacktrace:] con causa raíz
	java.lang.NumberFormatException: For input string: "123123345345"
		at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
		at java.base/java.lang.Integer.parseInt(Integer.java:652)
		at java.base/java.lang.Integer.parseInt(Integer.java:770)
		at org.apache.jsp.peticionesRegistro_jsp._jspService(peticionesRegistro_jsp.java:141)
		at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:777)
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:466)
		at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:380)
		at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:328)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:777)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:223)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:185)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:541)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:119)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:353)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:382)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:870)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1699)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:834)
03-Oct-2021 16:49:23.741 SEVERE [http-nio-8080-exec-26] org.apache.catalina.core.StandardWrapperValve.invoke El Servlet.service() para el servlet [jsp] en el contexto con ruta [] lanzó la excepción [Ha sucedido una excepción al procesar la página JSP [/peticionesRegistro.jsp] en línea [24]

21:         r.setEstado(Integer.parseInt(request.getParameter("estado")));
22:         r.setNombre_propietario(request.getParameter("nombre_propietario"));
23:         r.setDoc_propietario(Integer.parseInt(request.getParameter("doc_propietario")));
24:         r.setCel_propietario(Integer.parseInt(request.getParameter("cel_propietario")));
25:         r.setPlaca_vehiculo(request.getParameter("placa_vehiculo"));
26:         if(r.guardarRegistro()){
27:             respuesta += "\"" + proceso + "\": true";


Stacktrace:] con causa raíz
	java.lang.NumberFormatException: For input string: "123123345345"
		at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
		at java.base/java.lang.Integer.parseInt(Integer.java:652)
		at java.base/java.lang.Integer.parseInt(Integer.java:770)
		at org.apache.jsp.peticionesRegistro_jsp._jspService(peticionesRegistro_jsp.java:141)
		at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:777)
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:466)
		at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:380)
		at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:328)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:777)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:223)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:185)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:541)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:119)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:353)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:382)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:870)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1699)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:834)
03-Oct-2021 16:57:24.420 SEVERE [http-nio-8080-exec-24] org.apache.catalina.core.StandardWrapperValve.invoke El Servlet.service() para el servlet [jsp] en el contexto con ruta [] lanzó la excepción [Ha sucedido una excepción al procesar la página JSP [/peticionesRegistro.jsp] en línea [24]

21:         r.setEstado(Integer.parseInt(request.getParameter("estado")));
22:         r.setNombre_propietario(request.getParameter("nombre_propietario"));
23:         r.setDoc_propietario(Integer.parseInt(request.getParameter("doc_propietario")));
24:         r.setCel_propietario(Integer.parseInt(request.getParameter("cel_propietario")));
25:         r.setPlaca_vehiculo(request.getParameter("placa_vehiculo"));
26:         if(r.guardarRegistro()){
27:             respuesta += "\"" + proceso + "\": true";


Stacktrace:] con causa raíz
	java.lang.NumberFormatException: For input string: "12312313123"
		at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
		at java.base/java.lang.Integer.parseInt(Integer.java:652)
		at java.base/java.lang.Integer.parseInt(Integer.java:770)
		at org.apache.jsp.peticionesRegistro_jsp._jspService(peticionesRegistro_jsp.java:141)
		at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:777)
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:466)
		at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:380)
		at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:328)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:777)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:223)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:185)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:541)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:119)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:353)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:382)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:870)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1699)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:834)
03-Oct-2021 17:03:01.212 SEVERE [http-nio-8080-exec-33] org.apache.catalina.core.StandardWrapperValve.invoke El Servlet.service() para el servlet [jsp] en el contexto con ruta [] lanzó la excepción [Ha sucedido una excepción al procesar la página JSP [/peticionesRegistro.jsp] en línea [37]

34:         r.setId_registro_parqueo(Integer.parseInt(request.getParameter("id_registro")));
35:         r.setFecha_salida(request.getParameter("fecha_salida"));
36:         r.setEstado(Integer.parseInt(request.getParameter("estado")));
37:         r.setTiempo_parqueo(Integer.parseInt(request.getParameter("tiempo_parqueo")));
38:         r.setValor_parqueo(Integer.parseInt(request.getParameter("valor_parqueo")));
39:         if(r.actualizarRegistro()){
40:             respuesta += "\"" + proceso + "\": true";  // el \ se usa para concatenar en json indicando que se hizo el proceso (true)


Stacktrace:] con causa raíz
	java.lang.NumberFormatException: For input string: "NaN"
		at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
		at java.base/java.lang.Integer.parseInt(Integer.java:652)
		at java.base/java.lang.Integer.parseInt(Integer.java:770)
		at org.apache.jsp.peticionesRegistro_jsp._jspService(peticionesRegistro_jsp.java:154)
		at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:777)
		at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:466)
		at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:380)
		at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:328)
		at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:777)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:223)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:185)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:158)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:541)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:119)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:353)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:382)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:870)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1699)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:834)
03-Oct-2021 19:22:31.862 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextDestroyed()
03-Oct-2021 19:22:31.862 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextDestroyed()
